vms/platformvm/state/diff.go

```diff
@@ -572,23 +572,23 @@ func (d *diff) Apply(baseState Chain) error {
 			baseState.DeleteExpiry(entry)
 		}
 	}
-	// Ensure that all sov deletions happen before any sov additions. This
+	// Ensure that all l1validator deletions happen before any l1validator additions. This
 	// ensures that a subnetID+nodeID pair that was deleted and then re-added in
 	// a single diff can't get reordered into the addition happening first;
 	// which would return an error.
-	for _, sov := range d.sovDiff.modified {
-		if !sov.isDeleted() {
+	for _, l1validator := range d.l1ValidatorsDiff.modified {
```

Comment:
(no action) related to @marun comment about camel casing:
`l1validator`
`l1ValidatorsDiff`